reporter/src/funTest/kotlin/reporters/OpossumReporterFunTest.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```

Comment:
Why this Copyright? Did you copy a lot of HERE code?

Reply:
I copied the whole files with header and modified them. Without the removal of the copyright, since I am not sure whether IP from HERE is still in the code.